src/main/java/com/hotels/service/tracing/zipkintohaystack/ingresses/rabbitmq/RabbitMqIngressConfigProperties.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2018 Expedia, Inc.
+ *
+ *       Licensed under the Apache License, Version 2.0 (the "License");
+ *       you may not use this file except in compliance with the License.
+ *       You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *       Unless required by applicable law or agreed to in writing, software
+ *       distributed under the License is distributed on an "AS IS" BASIS,
+ *       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *       See the License for the specific language governing permissions and
+ *       limitations under the License.
+ *
+ */
+package com.hotels.service.tracing.zipkintohaystack.ingresses.rabbitmq;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("pitchfork.ingress.rabbitmq")
+public class RabbitMqIngressConfigProperties {
```

Comment:
Minor but since it's `RabbitMQ` can we capitalize the `q` whenever possible?

Reply:
Acronyms are usually an exception to capitalisation in CamelCase. For example, in Spring you have XmlReaderContext and not XMLReaderContext.